lib/mbta.rb

```diff
@@ -0,0 +1,31 @@
+origin_line
+origin_stop
+dest_line
+dest_stop
```

Comment:
These are all undefined at this point. This will cause errors when you run the program.